spec/models/donation_page_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe DonationPage, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:slug) }
+  it { should validate_presence_of(:person) }
+  it do
+    # uniqueness validation test was failing due to null constraint on person_id in db
+    # found workaround here: https://github.com/thoughtbot/shoulda-matchers/issues/194#issuecomment-31434217
+    create(:donation_page)
+    should validate_uniqueness_of(:slug).case_insensitive
+  end
+
+  describe ".by_funds_raised" do
+    it "lists donation pages by funds raised (desc)" do
+      worst_page = create(:donation_page)
+      best_page = create(:donation_page)
+      ok_page = create(:donation_page)
+      create(:action, donation_page: worst_page, donation_amount_in_cents: 100)
+      create(:action, donation_page: ok_page, donation_amount_in_cents: 300)
+      create(:action, donation_page: best_page, donation_amount_in_cents: 200)
+      create(:action, donation_page: best_page, donation_amount_in_cents: 200)
+
+      results = DonationPage.by_funds_raised
+
+      expect(results).to eq [best_page, ok_page, worst_page]
```

Comment:
I tried running `expect(results.count).to eq 3` and I got a weird postgres error.

```
SELECT COUNT(donation_pages.id, donation_pages.slug, donation_pages.visible_user_name, sum(actions.donation_amount_in_cents) as funds_raised_in_cents) AS count_donation_pages_id_donation_pages_slug_donation_pages_visi, donation_pages.id, donation_pages.slug, donation_pages.visible_user_name AS donation_pages_id_donation_pages_slug_donation_pages_visible_us FROM "donation_pages" INNER JOIN "actions" ON "actions"."donation_page_id" = "donation_pages"."id" GROUP BY donation_pages.id, donation_pages.slug, donation_pages.visible_user_name  ORDER BY funds_raised_in_cents desc
PG::SyntaxError: ERROR:  syntax error at or near "as"
LINE 1: ..._user_name, sum(actions.donation_amount_in_cents) as funds_r...
                                                             ^
```

Does it matter if we can't call `count` on this scope?